src/componentManifest.test.ts

```diff
@@ -148,7 +148,7 @@ describe('when there is a device entry point present', () => {
       ...makeClockfaceProjectConfig(),
       enableProposedAPI: true,
     } as ProjectConfiguration;
-    expectManifestJSON(
```

Comment:
This was a bug, the promise was dangling resulting in the test assertion going into the next test ran. The snapshot change reflects the correct result.